public/utils/config-equivalences.js

```diff
@@ -1,5 +1,9 @@
 export const configEquivalences = {
   pattern: 'Default index pattern to use on the app.',
+  'customization.logo.app':'Define the name of the app logo saved in the path /plugins/wazuh/assets/',
+  'customization.logo.sidebar':'Define the name of the sidebar logo saved in the path /plugins/wazuh/assets/',
+  'customization.logo.healthcheck':'Define the name of the health-check logo saved in the path /plugins/wazuh/assets/',
+  'customization.logo.reports':'Define the name of the reports logo (.png) saved in the path /plugins/wazuh/assets/',
```

Comment:
couldn't it be any type of image (png,svg,jpg,etc)?

Reply:
Currently, if the `.png` format is not used, it breaks when generating the report.